homeassistant/components/litterrobot/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "litterrobot",
+  "name": "Litter-Robot",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/litterrobot",
+  "requirements": ["pylitterbot==2021.2.5"],
+  "ssdp": [],
+  "zeroconf": [],
+  "homekit": {},
+  "dependencies": [],
```

Comment:
```suggestion
```

Reply:
Is this discoverable by `dhcp` ?

Reply:
It uses a WiFi module that is developed by Espressif, so unfortunately, I don't believe so.